doc/reference/tooling/tt_cli/tt_interactive_console.rst

```diff
@@ -0,0 +1,261 @@
+.. _tt-interactive-console:
+
+tt interactive console
+======================
+
+The ``tt`` utility features an command-line console that allows executing requests
+and Lua code interactively on the connected Tarantool instances.
+It is similar to the :ref:`Tarantool interactive console <interactive_console>` with
+one key difference: the ``tt`` console allows connecting to any available instance,
+both local and remote. Additionally, it offers more flexible output formatting capabilities.
```

Comment:
> both local and remote

We have the [Security](https://www.tarantool.io/en/doc/latest/book/admin/security/) topic (I don't know why it has such a name) that describes the difference between local and remote connections. Most of the contnet might be dropped but the main points are:
- A local connection (`tt connect <instance_name>`) opens a console for the `admin` user.
- A remote connection (`tt connect <iproto_socket/address>`) requires credentials for administration. Otherwise, the `guest` user is used.

So, I'd add the link to this topic or added a couple of sentences about this difference to the `Entering the console` section.